lib/selftests/Makefile

```diff
@@ -9,14 +10,16 @@ BPF_CFLAGS=-O2 -g -target bpf -D__TARGET_ARCH_x86 -mcpu=v3 -Wall
 BPF_CFLAGS+=-I/usr/include/bpf -I/usr/include/$(shell uname -m)-linux-gnu 
 BPF_CFLAGS+=$(INCLUDES)
 
+CC=/usr/bin/clang
```

Comment:
I think you can specify `CC=clang` and rely on the users $PATH as users may have clang installed elsewhere.